test/config-map.json

```diff
@@ -374,5 +374,13 @@
       "centos-10",
       "centos-9"
     ]
+  },
+  "./configs/modularity.json": {
+    "image-types": [
+      "qcow2"
```

Comment:
Please use `ami`, because only AWS images are boot-tested, so the `base-host-check.sh` script does not get run for `qcow2` :facepalm:

Reply:
Wow, thanks :)

Reply:
Done.